core/src/main/java/hudson/slaves/Cloud.java

```diff
@@ -310,8 +317,68 @@ public HttpResponse doDoDelete() throws IOException {
         return new HttpRedirect("..");
     }
 
+
+    @Override
+    public boolean isNameEditable() {
+        return false;
```

Comment:
shouldn't this default to true? 

i.e. this shows up for azure clouds:

![image](https://github.com/jenkinsci/jenkins/assets/21194782/8e28318c-ce6b-449f-8635-4af711946b37)

If the name isn't editable then shouldn't there be no rename page as well?

Reply:
So I tried hiding the sidepanel link, but there is code in the sidepanel jelly that will iterate through all the actions and displays the rename page regardless
https://github.com/jenkinsci/jenkins/blob/95373a96f710874d2b9d2f9bd40d28f1af7dc9e5/core/src/main/resources/hudson/slaves/Cloud/sidepanel.jelly#L36

Reply:
Actually, I stand corrected. When `isNameEditable` is false, there is no `Rename` Action and thus no link shows 😛